### PR TITLE
Disble wayland scaling workaround by default

### PR DESCRIPTION
**Pull request by AndreasBilke**
Add command line option to enable workaround behavor for affected
wayland sessions.

### CODE REVIEW

File: src/pdfpc.vala [hunk 1]
```diff
@@ -193,6 +194,10 @@ namespace pdfpc {
                 GLib.printerr("--notes option detected. Disable auto grouping.\n");
             }
 
+            if (Pdfpc.is_Wayland_backend() && !Options.wayland_workaround) {
+                GLib.printerr("If you have wrong scaling of the windows, consider using --wayland-workaround\n");
```
Comment (phillipberndt) [inline src/pdfpc.vala]: I'd place this into the documentation rather than outputting it every time pdfpc is started.

Comment (AndreasBilke) [reply src/pdfpc.vala]: I know that the man page might be the place, but do you think people will read it if they encounter a problem?

Comment (phillipberndt) [reply src/pdfpc.vala]: They won't, true. We could add an FAQ to SUPPORT.md, or stick with the current solution, I don't feel strongly about it.

Comment (fnevgeny) [reply src/pdfpc.vala]: I vote for a FAQ. IMHO, repeated warnings will be annoying if non-relevant. One can implement a one-time warning mechanism, but it isn't worth the trouble...

**Comment by AndreasBilke** [comment]
I added your suggestions. Can I merge it?

**Pull request merged**